### PR TITLE
removed create & update defs

### PR DESCRIPTION
**Pull request by jokirby**
# Description
Add description about what this pull request covers.
Removed unnecessary create & update defs. Functionality works fine w/o. 

## Number of Fixes
Type of number of fixes included in this PR.
1

## Related Ticket(s)
List related ticket(s) that apply to the work done in this pull request 

## Problem to Solve
Describe the problem(s) this pull request solves.
Removing unnecessary code.

## Expected Behavior
Describe the expected behavior.
Same functionality, less code.

## Steps to Test Solution

1. git fetch 
2. git checkout jk-remove-create-update
3. glance over code
4. python manage.py runserver

## Testing

[ ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[ ] I certify that all existing tests pass

## Documentation
[ ] There is new documentation in this pull request that must be reviewed..
[ ] I added documentation for any new classes/methods

## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

### CODE REVIEW

File: bangazon/bangazonapi/serializers.py [hunk 1]
```diff
@@ -21,27 +21,6 @@ class Meta:
...
-        ''' This method creates a new entry in the Products table
-
-        Argument List:
-            -validated_data: This argument how you pass in the data for the new table entry
-        '''
-        return Product.objects.create(**validated_data)
-
-    def update(self, instance, validated_data):
-        ''' This method creates a new entry in the Products table
-
```
Comment (stevenwally) [inline bangazon/bangazonapi/serializers.py:34]: nice removal

File: bangazon/bangazonapi/serializers.py [hunk 2]
```diff
@@ -21,27 +21,6 @@ class Meta:
...
-        Argument List:
-            -instance: This arguement is needed to update rather than replace data
-            -validated_data: This argument how you pass in the data for the new table entry
-        '''
-        instance.title = validated_data.get('title', instance.title)
```
Comment (stevenwally) [inline bangazon/bangazonapi/serializers.py:39]: exquisite removal

File: bangazon/bangazonapi/serializers.py [hunk 3]
```diff
@@ -55,28 +34,6 @@ class Meta :
         model = Payment_Type
         fields = '__all__'
 
-    def create(self, validated_data):
-        ''' This method creates a new entry in the Products table
-
-        Argument List:
```
Comment (stevenwally) [inline bangazon/bangazonapi/serializers.py:61]: cool removal

**Review by stevenwally — APPROVED** [review]
Nice work. Same functionality, less code

👍

**Comment by nathantbaker** [comment]
👍  I fetched the branch and api functionality works as expected. I also see the code affected is only removing the methods we want. 🚀

**Pull request merged**